src/hub_predtimechart/app/generate_target_json_files.py

```diff
@@ -0,0 +1,114 @@
+import json
+from datetime import date, timedelta
+from pathlib import Path
+
+import click
+import polars as pl
+import structlog
+
+from hub_predtimechart.app.generate_json_files import json_file_name
+from hub_predtimechart.hub_config import HubConfig
+from hub_predtimechart.util.logs import setup_logging
+
+
+setup_logging()
+logger = structlog.get_logger()
+
+
+@click.command()
+@click.argument('hub_dir', type=click.Path(file_okay=False, exists=True))
+@click.argument('ptc_config_file', type=click.Path(file_okay=True, exists=False))
+@click.argument('target_out_dir', type=click.Path(file_okay=False, exists=True))
+def main(hub_dir, ptc_config_file, target_out_dir):
+    '''
+    Generates the target data json files used by https://github.com/reichlab/predtimechart to
+    visualize a hub's forecasts.
+
+    HUB_DIR: (input) a directory Path of a https://hubverse.io hub to generate target data json files from
+
+    TARGET_OUT_DIR: (output) a directory Path to output the viz target data json files to
+    \f
+    :param hub_dir: (input) a directory Path of a https://hubverse.io hub to generate target data json files from
+    :param target_out_dir: (output) a directory Path to output the viz target data json files to
```

Comment:
This should be updated to reflect the new arguments.

Reply:
Thanks!